content/md/supplier-onboarder/handle-category.md

```diff
@@ -0,0 +1,118 @@
+---
+id: handle-category
+themes: manage-products
+title: Handle **categories**
+popular: false
+---
+
+# Definitions
+## Definition of a family
+In Akeneo Onboarder, a product can only be defined by *one* family and its specific set of attributes. A family **defines the structure of products**.
+
+:::info
+The [attributes](./onboarder/articles/update-products.html#the-attribute-panel) are the different pieces of information that you'll have to provide.
+:::
+
+## Definition of a category
+In Akeneo Onboarder, you can create a category tree to help you to sort your products and find them. The categories are how you **classify a product**. A category is always part of a category tree (or *classification tree*).
+
+As an Onboarder user, you can only have **one category tree** in Akeneo with an **unlimited number of levels** (categories, subcategories, sub-subcategories).
+
+You can classify a product in one or several categories. It's possible for a product to not be in a category, in that case, it goes to the 'Unclassified' Category by default.
+
+![image of the category trees](../img/Settings_CategoryTree.png)
+
+
+
+## Differences between families and categories
+In this section, we'll take an example to compare two key concepts of the Akeneo Onboarder: *Families* and *Categories*.
+
+Our catalog contains lots of furnitures: chairs and tables for instance.
+On the one hand, for chairs you'll define: heigh, material, seat depth, foam softness, etc, And, for table you'll define heigh, length, with of the board, number of feet, etc. You see that the two products don't have the same properties, it means that the sets of attributes are different. Finally, they belong to separated **families**.
+
+On the other hand, for all kind of furnitures, you can set them in the indoor or outdoor section of your catalog by using a **category**. In our example, the chairs and tables can be one, both or none of the categories.
```

Comment:
@camillebourjade following your suggestion, I added an example here.

Reply:
Then, all of your furniture can belong to the indoor or outdoor section of your catalog, which is structured by a **category**. In our example, the chairs and tables can be one, both, or none of the categories.